src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -12,7 +12,9 @@ define([
         interval: 30000,
         once: false,
         startatonce: true,
+        callEvent: null,
```

Comment:
change `callEvent` type to `"callMicroflow" | "callNanoflow";`

Reply:
done